pkg/node/local.go

```diff
@@ -208,9 +208,13 @@ func StartLocalNode(
 		avalancheGoVersion, err = GetAvalancheGoVersion(app, avaGoVersionSetting)
 		if err != nil {
 			return err
-		} else {
-			ux.Logger.PrintToUser("Using AvalancheGo version: %s", avalancheGoVersion)
 		}
+		_, avagoDir, err := binutils.SetupAvalanchego(app, avalancheGoVersion)
```

Comment:
nit: I checked all usage of `binutils.SetupAvalanchego` and avagoDir is always used to only to get path to avalanchego. Can we change this func to return path to avalanchego right away and not dir ?